docs/manual/docs/cmd/teams/report/report-useractivityuserdetail.md

```diff
@@ -29,4 +31,9 @@ Gets details about Microsoft Teams user activity by user for July 13, 2019
 
 ```sh
 teams report useractivityuserdetail --date 2019-07-13
+```
+Gets details about Microsoft Teams user activity by user for the last week and exports the report data in the specified path in csv format
+
+```sh
+teams report useractivityuserdetail --period D7 --outputFile '/Users/josephvelliah/Desktop/teams-report-useractivityuserdetail.csv'
```

Comment:
Seems like you've build functionality for `json` export, but your example is with `csv`. Could you please align?

Reply:
I have fixed this in the latest commit.